frontend/src/scenes/insights/filters/ActionFilter/RenameModal.tsx

```diff
@@ -6,6 +6,7 @@ import { Button, Input, Modal } from 'antd'
 import { getDisplayNameFromEntityFilter } from 'scenes/insights/utils'
 import { renameModalLogic } from 'scenes/insights/filters/ActionFilter/renameModalLogic'
 import { InputFocusOptions } from 'antd/lib/input/Input'
+import InputRef from 'antd/lib/input/Input'
```

Comment:
Is it better to replace the antd input instead?

Reply:
Yep, done in https://github.com/PostHog/posthog/pull/18242